### PR TITLE
Enable PHPStan on all controllers

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | PHPStan was configured to run only on the legacy controller. This PR changes the configuration 
| Type?             | improvement
| BC breaks?        | Nope
| Deprecations?     | Nope
| Fixed ticket?     | /
| Sponsor company   | @PrestaShopCorp
| How to test?      | CI must be green once the errors from PHPStan are fixed.

### CODE REVIEW

**Review by ga-devfront — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
No need of QA review for this one, PHPStan covers it.

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
